packages/cli/src/cliEntry.js

```diff
@@ -79,7 +79,7 @@ function printHelpInformation(examples, pkg) {
     output = output.concat([chalk.bold('\nExample usage:'), formattedUsage]);
   }
 
-  return output.join('\n');
+  return output.concat('').join('\n');
```

Comment:
How about 
```suggestion
  return output.join('\n').concat('\n');
```
which would make it more obvious that we want to add a final newline?

Reply:
I think the way you said it is better. Thanks :)